### ISSUE TITLE
Can plugins be approved please?

### ISSUE THREAD

**Issue by cyberpwnn** [Hidendra/mcstats.org#35]
I'm waiting on a plugin called Phantom here http://mcstats.org/plugin/Phantom
I can see it's loading some data, and i have claimed it, however

`Your ownership request for Phantom is still pending approval, which was submitted at 23:44 UTC Fri, September 02`

**Comment by sgdc3** [comment]
@cyberpwnn Sadly this project is dying, @Hidendra is apparently busy with his personal life, the only thing that he can do is to restart the service when it crashes.
Due to this reason me and other devs started @StatsCraft-MC , but in order to continue the development we need more developers, would you be interested? ;)

**Comment by cyberpwnn** [comment]
It is interesting definitely, but i have been really busy with hammering out plugins and minigames for a network. However i will be checking on that project to see how it turns out :P

Yeah, i think its been dying for the past year or so. I just wish it was automatic. Anyways thanks for the quick response. I look forward to StatsCraft-MC!

**Comment by sgdc3** [comment]
@cyberpwnn Thank you for the support :)

**Comment by jamierocks** [comment]
@sgdc3 while I can agree that another stats platform because mcstats isn't so well maintained is a good thing BUT using mcstats' issue tracker to gain traction for your alternative/competitor IMO is wrong. I wouldn't go to Spigot's issue tracker and tell everyone to use Sponge, even if I thought they should.

**Comment by sgdc3** [comment]
@jamierocks Our aim is to provide a working project, there is nothing commercial in that, so in my opinion is fine.

**Comment by sgdc3** [comment]
Spigot and Sponge are both active project, MCStats is unmaintained since months!

**Comment by jamierocks** [comment]
I couldn't care any less that MCStats is 'unmaintained', but I'm not part of the MCStats team so my word isn't final here. Regardless if I was I wouldn't be letting someone advertise their up and coming project. Like I said this isn't the place for you to advertise your product.

**Comment by sgdc3** [comment]
I don't want to create a "product", our project is open to anyone and without support it has no chances to see the daylight.

**Comment by jamierocks** [comment]
Like I said I'm all for another plugin statistics site (I was all for https://github.com/Statik-Metrics), but using the popularity of another still running 'project' is wrong. -_-

**Comment by sgdc3** [comment]
@jamierocks I'm doing something despite you, i don't want to "steal" mcstats popularity, i only want a working stats service made for free and 100% opensource. Nothing else.

**Comment by cyberpwnn** [comment]
If i could make a point... I was never aware of @StatsCraft-MC. Now that i know about it, sure, i probably wont be looking into MCstats much more. MCStats is dead. I dont think there is theft of MCstats popularity. Its just the shared interest of keeping statistics on plugins. 

Also, Spigot is well alive, not dead. So sure, broadcasting sponge would not be a good idea. However if spigot was dying, and someone asked a question similar to mine about spigot, someone mentioning sponge would probably be more helpful than "stealing popularity of the past" from a dead project.

**Comment by jamierocks** [comment]
This is an issue tracker. Sure if you were talking about this on MCStat's forum (if it had one) sure that'd be cool. Let's keep this issue tracker on point please. -_-

**Comment by sgdc3** [comment]
Very useful an issue tracker for a dead project

**Comment by cyberpwnn** [comment]
I closed the issue as soon as i was finished with it. It's just banter at this point. Which i'm fine with. MCStats is dead. This issue is closed. It's not cluttering the tracker. 

The project is dead. No need to pour salt all over everyone who sways off topic of mcstats while remaining on topic on minecraft server plugin statistics.

**Comment by jamierocks** [comment]
Is this the issue tracker for 'Minecraft server plugin statistics'? No it's not it's mcstats' issue tracker. If hidendra was still around I'm certain this issue would be locked and limited to contributors.
The actual issue here (long delays in plugins getting approved) hasn't actual been resolved.
And I think this misconception of the project being dead needs to be ended, this has had no announcement! Infact (although sometimes with quite the delay) hidendra has been keeping the site up.

**Comment by sgdc3** [comment]
@jamierocks keeping the site up = restarting the backend after months of downtime

**Comment by jamierocks** [comment]
MCStats has never been down for 'months'. -_-

**Comment by sgdc3** [comment]
I remember very well, other devs can confirm that.

**Comment by sgdc3** [comment]
I love mcstats as project, but you are blind if you think it's still active.

**Comment by sgdc3** [comment]
It is more like a zombie

**Comment by jamierocks** [comment]
Did I say it was active? No. I said it's not dead. I said that this is an issue tracker for MCStats, and stated that it should not be used like a forum. I am all for alternatives to MCStats (although TBH I've yet to see much coming from your project), but this issue tracker (or any issue tracker that isn't your own) is not the place to advertise your project.

**Comment by sgdc3** [comment]
I only want to inform people and maybe convince  someone to collaborate to our project. I want to talk directly to developers not to kids on forums like on spigotmc.org

**Comment by jamierocks** [comment]
Then do so in relevant places.

EDIT: I want to clarify my position here: I can't wait to see what happens with StatsCraft, but at the same time I don't want to see MCStats' issue tracker have issues closed when they're not resolved. (An alternative is not the solution to this issue for example.). I wish you the best with your endeavours ;)
